tsp-output/@azure-tools/typespec-csharp/src/Generated/OpenAIClient.cs

```diff
@@ -46,100 +46,20 @@ public OpenAIClient(Uri endpoint, ApiKeyCredential credential, OpenAIClientOptio
             options ??= new OpenAIClientOptions();
 
             _keyCredential = credential;
-            _pipeline = ClientPipeline.Create(options, Array.Empty<PipelinePolicy>(), new PipelinePolicy[] { ApiKeyAuthenticationPolicy.CreateHeaderApiKeyPolicy(_keyCredential, AuthorizationHeader, AuthorizationApiKeyPrefix) }, Array.Empty<PipelinePolicy>());
+            var authenticationPolicy = ApiKeyAuthenticationPolicy.CreateBearerAuthorizationPolicy(_keyCredential);
```

Comment:
I don't know what the TypeSpec indicates here, but I believe we had decided OpenAI should use this factory method.  I don't feel super strongly about this, but IIRC we had tried to align constructor names to OpenAI auth options, so it might be at least doing due-diligence on that for TypeSpec completeness.

Reply:
https://github.com/Azure/autorest.csharp/issues/4489